roles/bootstrap-os/tasks/bootstrap-fedora.yml

```diff
@@ -44,3 +44,14 @@
   become: true
   when:
     - not is_atomic
+  with_items:
```

Comment:
This loop seems not to do anything in this position

Reply:
Good catch. Looks like things got discombobulated when resolving merge conflicts across the ages.